.travis.yml

```diff
@@ -0,0 +1,47 @@
+sudo: required
+
+env:
+  - service=11.0.5-jre-slim
+
+  global:
+    - DOCKER_COMPOSE_VERSION=1.22.0
+    - DOCKER_IMAGE=wma-docker-java
+
+services:
+  - docker
+
+before_install:
+  - sudo apt-get update
+  - sudo rm /usr/local/bin/docker-compose
+  - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+  - chmod +x docker-compose
+  - sudo mv docker-compose /usr/local/bin
+  - sudo apt-get -y -o Dpkg::Options::="--force-confnew" install docker-ce
+  - curl -L https://github.com/arminc/clair-scanner/releases/download/v8/clair-scanner_linux_amd64 > clair-scanner
+  - chmod +x clair-scanner
+  - sudo mv clair-scanner /usr/local/bin
+  - git clone https://github.com/docker/docker-bench-security.git
+
+install:
+  - docker-compose up --detach --build $service
```

Comment:
How;s this work if there is no compose configuration or healthcheck internal to the container?